.vscode/launch.json

```diff
@@ -64,87 +61,87 @@
             "type": "python",
             "request": "launch",
             "program": "pbsync/pbsync.py",
-            "console": "integratedTerminal",
+            "console": "integratedTerminal"
         },
         {
             "name": "Sync All",
```

Comment:
--debugbranch parameters is broken after those code changes.

Expected output (from master branch):
```
10-May-20 18:45:34 [INFO ]  Fetching recent changes on the repository...
10-May-20 18:45:41 [INFO ]  ------------------
10-May-20 18:45:42 [INFO ]  On branch code/pbsync-updates
Your branch is up to date with 'origin/code/pbsync-updates'.

nothing to commit, working tree clean
10-May-20 18:45:42 [INFO ]  Please wait while getting latest changes on the repository. It may take a while...
Branch 'code/pbsync-updates' set up to track remote branch 'code/pbsync-updates' from 'origin'.
10-May-20 18:45:42 [INFO ]  Trying to stash the local work...
10-May-20 18:45:43 [INFO ]  No local changes to save
10-May-20 18:45:43 [INFO ]  Trying to rebase workspace with latest changes on the repository...
10-May-20 18:45:46 [INFO ]  Current branch code/pbsync-updates is up to date.
10-May-20 18:45:46 [INFO ]  Trying to pop stash...
10-May-20 18:45:46 [INFO ]  No stash entries found.
10-May-20 18:45:46 [INFO ]  Success, rebased on latest changes without any conflict
10-May-20 18:45:46 [INFO ]  ------------------
10-May-20 18:45:46 [INFO ]  Current project version: 0.63.22
```

Output from this PR:
```
10-May-20 18:47:44 [INFO ]  Fetching recent changes on the repository...
10-May-20 18:47:56 [INFO ]  ------------------
10-May-20 18:47:56 [WARNI]  Current branch is not supported for repository synchronization: code/pbsync-updates. Auto synchronization will be disabled
10-May-20 18:47:56 [INFO ]  ------------------
10-May-20 18:47:56 [INFO ]  Checking for engine updates...
```